### PR TITLE
Geochronology page updates

### PR DESCRIPTION
**Pull request by ZachBeck**
Additional dating techniques added. Hub downloads updated.

### CODE REVIEW

File: data/geoscience/geochronology/index.html [hunk 1]
```diff
@@ -53,7 +89,7 @@ <h3 id="Argon">Argon</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{% link images/argon.png %}" alt="Argon" />
-    <p>Argon: Argon–Argon (40Ar/39Ar) dating is a relative radiometric dating method that relies on neutron irradiation from a nuclear reactor to convert a stable form of potassium (39K) into the radioactive 39Ar, is primarily used on metamorphic and igneous minerals, and generally replaces the older potassium-argon (K/Ar) method.  The Geoscience.GeochronArgon data layer represents the argon (40Ar/39Ar) dating data in the Utah Geochronology Database.</p>
+    <p>Argon: Argon–Argon (40Ar/39Ar) dating is a relative radiometric dating method that relies on neutron irradiation from a nuclear reactor to convert a stable form of potassium (39K) into the radioactive 39Ar, is primarily used on metamorphic and igneous minerals, and generally replaces the older potassium-argon (K/Ar) method.  The Utah Geochronology Argon feature layer represents the argon (40Ar/39Ar) dating data in the Utah Geochronology Database.</p>
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:92]: **nitpick**: remove double space
```suggestion
    <p>Argon: Argon–Argon (40Ar/39Ar) dating is a relative radiometric dating method that relies on neutron irradiation from a nuclear reactor to convert a stable form of potassium (39K) into the radioactive 39Ar, is primarily used on metamorphic and igneous minerals, and generally replaces the older potassium-argon (K/Ar) method. The Utah Geochronology Argon feature layer represents the argon (40Ar/39Ar) dating data in the Utah Geochronology Database.</p>
```

File: data/geoscience/geochronology/index.html [hunk 2]
```diff
@@ -19,6 +19,18 @@
     item_id: de953b38d0df43468bb2b1947697a508
   updates:
     - August 2020
+Cosmogenic:
+  hub:
+    name: Utah Geochronology Cosmogenic 
+    item_id: 8c2f4e18e28a44af96d2f57d88dbd37c
+  updates:
+    - August 2020
+Fission Track:
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:28]: **chore**: I'm pretty sure these cannot contain spaces

File: data/geoscience/geochronology/index.html [hunk 3]
```diff
@@ -66,12 +102,50 @@ <h5>Related Resources</h5>
...
+    <p>Fission Track: Fission Track dating is a radiometric dating technique that analyizes the damage of tracks/trails left by fission fragments from the decay of uranium-238. The Utah Geochronology Fission Track feature layer represents the fission track dating data in the Utah Geochronology Database.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://geology.utah.gov/apps/geochron/">Utah Geochronology Database</a>
+    </ul>
+    <p></p>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.geochronology %}</p>
+  </div>
+  {% include packagedata.html
+    name="FissionTrack"
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:139]: **chore**: this needs to match the front matter label. This is using the correct naming convention.

File: data/geoscience/geochronology/index.html [hunk 4]
```diff
@@ -88,7 +162,7 @@ <h3 id="Luminescense">Luminescense</h3>
...
+    <p>Luminescence: Luminescence dating refers to a group of methods, including optically stimulated luminescence (OSL), infrared stimulated luminescence (IRSL), and thermoluminescence dating (TL), that determine how long ago mineral grains were last exposed to sunlight or sufficient heating.  It is commonly used by geologists and archaeologists to determine when that event occurred, such as surface fault rupture from an earthquake.  Optical dating typically refers to OSL and IRSL, but not TL. The Geochronology Luminescence feature layer represents the luminescence (OSL, IRSL, and TL) dating data in the Utah Geochronology Database.</p>
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:165]: **nitpick**: remove double spaces
```suggestion
    <p>Luminescence: Luminescence dating refers to a group of methods, including optically stimulated luminescence (OSL), infrared stimulated luminescence (IRSL), and thermoluminescence dating (TL), that determine how long ago mineral grains were last exposed to sunlight or sufficient heating. It is commonly used by geologists and archaeologists to determine when that event occurred, such as surface fault rupture from an earthquake. Optical dating typically refers to OSL and IRSL, but not TL. The Geochronology Luminescence feature layer represents the luminescence (OSL, IRSL, and TL) dating data in the Utah Geochronology Database.</p>
```

File: data/geoscience/geochronology/index.html [hunk 5]
```diff
@@ -126,7 +200,7 @@ <h3 id="Radiocarbon">Radiocarbon</h3>
...
+    <p>Radiocarbon: Radiocarbon dating is a radiometric dating method for determining the age of an object containing organic material by using the properties of the radioactive isotope of carbon (14C). Measurement of radiocarbon was originally performed using beta-counting devices, which counted the amount of beta radiation emitted by decaying 14C atoms in a sample.  Accelerator mass spectrometry (AMS) is now used, and counts all of the 14C atoms in the sample, and not just the few that happen to decay during the measurement interval and can be used with significantly smaller samples (such as individual plant seeds), and gives results more quickly.  The Utah Geochronology Radiocarbon feature layer represents the radiocarbon (C or AMS) dating data in the Utah Geochronology Database.</p>
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:203]: **nitpick**: remove double spaces
```suggestion
    <p>Radiocarbon: Radiocarbon dating is a radiometric dating method for determining the age of an object containing organic material by using the properties of the radioactive isotope of carbon (14C). Measurement of radiocarbon was originally performed using beta-counting devices, which counted the amount of beta radiation emitted by decaying 14C atoms in a sample. Accelerator mass spectrometry (AMS) is now used, and counts all of the 14C atoms in the sample, and not just the few that happen to decay during the measurement interval and can be used with significantly smaller samples (such as individual plant seeds), and gives results more quickly. The Utah Geochronology Radiocarbon feature layer represents the radiocarbon (C or AMS) dating data in the Utah Geochronology Database.</p>
```

File: data/geoscience/geochronology/index.html [hunk 6]
```diff
@@ -139,3 +213,79 @@ <h5>Related Resources</h5>
...
+    info=page.RubidiumStrontium
+  %}
+</div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="Tephrochronology">Tephrochronology</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/lum.png %}" alt="Tephrochronology" />
+    <p>Tephrochronology: Tephrochronology is a dating technique that uses distinct layers of volcanic ash to create a geochronology in which objects or events can be dated relatively to the ash stratigraphy.  The Utah Geochronology Tephrochronology feature layer represents the tephrochronology dating data in the Utah Geochronology Database.</p>
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:241]: **nitpick**: remove double spaces
```suggestion
    <p>Tephrochronology: Tephrochronology is a dating technique that uses distinct layers of volcanic ash to create a geochronology in which objects or events can be dated relatively to the ash stratigraphy. The Utah Geochronology Tephrochronology feature layer represents the tephrochronology dating data in the Utah Geochronology Database.</p>
```

**Review by steveoh — CHANGES_REQUESTED** [review]
Wow thanks for all of these additions.

The frontmatter labels need to have spaces removed to match the packagedata.html include name. Otherwise this looks ready to merge!

File: data/geoscience/geochronology/index.html (hunk 3 above)
Comment (ZachBeck) [reply data/geoscience/geochronology/index.html:139]: done

**Review by ZachBeck — COMMENTED** [review]
should be good now

File: data/geoscience/geochronology/index.html [hunk 7]
```diff
@@ -33,6 +45,30 @@
     item_id: 3c3d19ed6a4c43ec8fb5d02d1dde37d4
   updates:
     - August 2020
+RubidiumStrontium:
+  hub:
+    name: Utah Geochronology Rubidium Strontium
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:50]: **issue**: this isn't the correct name or something is up since it is creating a 404 in the links.

**Review by steveoh — CHANGES_REQUESTED** [review]
I think you spelled `Utah Geochronology Rhubidium Strontium` incorrectly or something. Will you make sure all of the others are valid.

File: data/geoscience/geochronology/index.html [hunk 8]
```diff
@@ -139,3 +213,79 @@ <h5>Related Resources</h5>
...
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://geology.utah.gov/apps/geochron/">Utah Geochronology Database</a>
+    </ul>
+    <p></p>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.geochronology %}</p>
+  </div>
+  {% include packagedata.html
+    name="RubidiumStrontium"
+    info=page.RubidiumStrontium
```
Comment (steveoh) [inline data/geoscience/geochronology/index.html:232]: **chore**: all of these spellings are wrong

**Review by steveoh — CHANGES_REQUESTED** [review]
after part 3 i think this will be ready to go

**Pull request closed without merging**